changelogs/unreleased/add-check-for-user-input-of-the-number-of-iproto-threads.md

```diff
@@ -1,4 +1,4 @@
 ## bugfix/core
 
- * Added check for user input of the number of iproto threads - value
-   must be > 0 and less then or equal to 1000 (gh-6005).
+ * Added check for user input of the number of iproto threads — value
+   must be > 0 and less than or equal to 1000 (gh-6005).
```

Comment:
There is a thing around hyphen/dash, which doubts me.

There are technical texts, which aim direct reading (without any pre-rendering) as a primary or a second goal (say, markdown supports rendering, but easy to read directly). The most obvious example is a comment in code. All or most of text that are tightly connected to code are also of this kind: they are usually written as plain text, asciidoc, markdown. Here non-ASCII symbols are generally discouraged. Here we expect a hyphen (or two hyphens), where in book we would see a dash.

There are texts, which primarily aim some rendering tool, which will prettify it, inline hyperlinks and convert to a format of a reader tools (generate HTML, PDF and so on). Those are texts for modern websites for example. Here we use Unicode en/em dash symbols or a special syntax to represent them using ASCII.

I feel release notes as a text of the first kind. Is the neural network in my head wrong here? What do you think?

Reply:
I agree. So I am going to apply the rule from the doc [contribution guide](https://www.tarantool.io/ru/doc/latest/contributing/docs/style/#dashes) and use "---" as a dash. The documentation builder renders this character sequence in the source into em dash.